CarpetX/src/schedule.cxx

```diff
@@ -1822,7 +1858,7 @@ int Evolve(tFleshConfig *config) {
       total_evolution_output_time += output_finish_time - output_start_time;
 
       active_levels = optional<active_levels_t>();
```

Comment:
this is the one we need to encounter and why a "continue" must not be used? I wonder if this could be incorporated into the loop header itself.